.github/workflows/ci.yml

```diff
@@ -0,0 +1,16 @@
+name: ci
+on: pull_request
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        node-version: [8.x, 10.x, 12.x]
+    steps:
+    - uses: actions/checkout@v1
+    - name: Use Node.js
+      uses: actions/setup-node@v1
+      with:
+        node-version: ${{ matrix.node-version }}
+    - name: install and test
+      run: npm it
```

Comment:
what is `npm it`? The commands to be run include: `npm ci` to download deps, and then `npm run test` which will compile the code and then run the test suite.